src/pineko/fonll.py

```diff
@@ -235,7 +239,7 @@ def collect_updates(fonll_fns, damp):
     """Produce the different theory cards according to which FONLL is asked for."""
     updates = []
     is_mixed = fonll_fns in MIXED_ORDER_FNS
-    is_damped = damp != 0
+    is_damped = damp > 0
```

Comment:
Just to remind/clarify: in
https://github.com/NNPDF/pineko/issues/141#issuecomment-1837481519
I suggested not doing this...

I don't have such a strong opinion, so in the end just do whatever you wish.